docs/docs/config.md

```diff
@@ -51,6 +51,14 @@ proxy:
 # Aether support pulling data from multiple providers
 # Each provider has a set of configurations
 providers:
+  # Google Cloud Provider
```

Comment:
I notice we have some duplication with the example configs. Maybe we remove the example from the docs, and point to the `example.local.yaml`? Or vis-versa. But that way we aren't changing things in more than one place.

Reply:
The question here is if we want the configuration in our documentation or in our repo?

Reply:
hmm valid point. Maybe boht is good then. The documentation site can go more in depth